chain/types/ethtypes/eth_transactions.go

```diff
@@ -21,8 +19,35 @@ import (
 	"github.com/filecoin-project/lotus/chain/types"
 )
 
-const Eip1559TxType = 2
+const (
+	// LegacyHomesteadEthTxSignaturePrefix defines the prefix byte used to identify the signature of a legacy Homestead Ethereum transaction.
+	LegacyHomesteadEthTxSignaturePrefix = 0x01
+	Eip1559TxType                       = 0x2
 
+	ethLegacyHomesteadTxType    = 0x0
+	ethLegacyHomesteadTxChainID = 0x0
```

Comment:
```suggestion
	LegacyHomesteadEthTxSignaturePrefix = 0x01
	Eip1559TxType                       = 0x02

	ethLegacyHomesteadTxType    = 0x00
	ethLegacyHomesteadTxChainID = 0x00
```

or the other way, just for consistency